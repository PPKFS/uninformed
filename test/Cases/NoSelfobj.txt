Test me with "bob/privacy/look/bob/south".

Northside is north of Southside. Aaron, Bob, and Charlie
are men in Northside. Player is Aaron. 

Definition: a person is NPC if it is not the player. 

Following relates people to each other. The verb to be
following implies the following relation. 

The last carry out going: now every person following player
is in the location.

Understand "[person]" as toggling follow. Toggling follow
is an action applying to one thing. 

Carry out toggling follow:
	if noun is following player:
		now noun is not following player;
	otherwise:
		now noun is following player;
	say "[list of NPC people following player] will attend you.
	[list of NPC people not following player] will not."

Understand "privacy" as ostracizing. Ostracizing is an
action applying to nothing. Carry out ostracizing: now
every NPC person not following player is in a random room
which is not the location; say "[list of every NPC person
not following the player] leaves the room."
