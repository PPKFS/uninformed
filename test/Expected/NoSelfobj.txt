156 words
¶test¶ ¶me¶ ¶with¶ ¶"bob/privacy/look/bob/south"¶ ¶.¶ ¶|__¶ ¶northside¶ ¶is¶ ¶north¶ ¶of¶ ¶southside¶ ¶aaron¶ ¶,¶ ¶bob¶ ¶and¶ ¶charlie¶ ¶are¶ ¶men¶ ¶in¶ ¶player¶ ¶definition¶ ¶:¶ ¶a¶ ¶person¶ ¶npc¶ ¶if¶ ¶it¶ ¶not¶ ¶the¶ ¶following¶ ¶relates¶ ¶people¶ ¶to¶ ¶each¶ ¶other¶ ¶verb¶ ¶be¶ ¶implies¶ ¶relation¶ ¶last¶ ¶carry¶ ¶out¶ ¶going¶ ¶now¶ ¶every¶ ¶location¶ ¶understand¶ ¶"[person]"¶ ¶as¶ ¶toggling¶ ¶follow¶ ¶an¶ ¶action¶ ¶applying¶ ¶one¶ ¶thing¶ ¶noun¶ ¶;¶ ¶otherwise¶ ¶say¶ ¶"[list of NPC people following player] will attend you.
[list of NPC people not following player] will not."¶ ¶"privacy"¶ ¶ostracizing¶ ¶nothing¶ ¶random¶ ¶room¶ ¶which¶ ¶"[list of every NPC person
not following the player] leaves the room."¶ 
68 distinct words
¶Test¶ ¶test¶ 0a
¶me¶ ¶me¶ 20
¶with¶ ¶with¶ 20
¶"bob/privacy/look/bob/south"¶ ¶"bob/privacy/look/bob/south"¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶Northside¶ ¶northside¶ 20
¶is¶ ¶is¶ 20
¶north¶ ¶north¶ 20
¶of¶ ¶of¶ 20
¶Southside¶ ¶southside¶ 20
¶.¶ ¶.¶ 20
¶Aaron¶ ¶aaron¶ 20
¶,¶ ¶,¶ 20
¶Bob¶ ¶bob¶ 20
¶,¶ ¶,¶ 20
¶and¶ ¶and¶ 20
¶Charlie¶ ¶charlie¶ 20
¶are¶ ¶are¶ 0a
¶men¶ ¶men¶ 20
¶in¶ ¶in¶ 20
¶Northside¶ ¶northside¶ 20
¶.¶ ¶.¶ 20
¶Player¶ ¶player¶ 20
¶is¶ ¶is¶ 20
¶Aaron¶ ¶aaron¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶Definition¶ ¶definition¶ 20
¶:¶ ¶:¶ 20
¶a¶ ¶a¶ 20
¶person¶ ¶person¶ 20
¶is¶ ¶is¶ 20
¶NPC¶ ¶npc¶ 20
¶if¶ ¶if¶ 20
¶it¶ ¶it¶ 20
¶is¶ ¶is¶ 20
¶not¶ ¶not¶ 20
¶the¶ ¶the¶ 20
¶player¶ ¶player¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶Following¶ ¶following¶ 20
¶relates¶ ¶relates¶ 20
¶people¶ ¶people¶ 20
¶to¶ ¶to¶ 20
¶each¶ ¶each¶ 20
¶other¶ ¶other¶ 20
¶.¶ ¶.¶ 20
¶The¶ ¶the¶ 20
¶verb¶ ¶verb¶ 20
¶to¶ ¶to¶ 20
¶be¶ ¶be¶ 20
¶following¶ ¶following¶ 0a
¶implies¶ ¶implies¶ 20
¶the¶ ¶the¶ 20
¶following¶ ¶following¶ 20
¶relation¶ ¶relation¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶The¶ ¶the¶ 20
¶last¶ ¶last¶ 20
¶carry¶ ¶carry¶ 20
¶out¶ ¶out¶ 20
¶going¶ ¶going¶ 20
¶:¶ ¶:¶ 20
¶now¶ ¶now¶ 20
¶every¶ ¶every¶ 20
¶person¶ ¶person¶ 20
¶following¶ ¶following¶ 20
¶player¶ ¶player¶ 20
¶is¶ ¶is¶ 0a
¶in¶ ¶in¶ 20
¶the¶ ¶the¶ 20
¶location¶ ¶location¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶Understand¶ ¶understand¶ 20
¶"[person]"¶ ¶"[person]"¶ 20
¶as¶ ¶as¶ 20
¶toggling¶ ¶toggling¶ 20
¶follow¶ ¶follow¶ 20
¶.¶ ¶.¶ 20
¶Toggling¶ ¶toggling¶ 20
¶follow¶ ¶follow¶ 20
¶is¶ ¶is¶ 0a
¶an¶ ¶an¶ 20
¶action¶ ¶action¶ 20
¶applying¶ ¶applying¶ 20
¶to¶ ¶to¶ 20
¶one¶ ¶one¶ 20
¶thing¶ ¶thing¶ 20
¶.¶ ¶.¶ 20
¶|__¶ ¶|__¶ 0a
¶Carry¶ ¶carry¶ 20
¶out¶ ¶out¶ 20
¶toggling¶ ¶toggling¶ 20
¶follow¶ ¶follow¶ 20
¶:¶ ¶:¶ 20
¶if¶ ¶if¶ 41
¶noun¶ ¶noun¶ 20
¶is¶ ¶is¶ 20
¶following¶ ¶following¶ 20
¶player¶ ¶player¶ 20
¶:¶ ¶:¶ 20
¶now¶ ¶now¶ 42
¶noun¶ ¶noun¶ 20
¶is¶ ¶is¶ 20
¶not¶ ¶not¶ 20
¶following¶ ¶following¶ 20
¶player¶ ¶player¶ 20
¶;¶ ¶;¶ 20
¶otherwise¶ ¶otherwise¶ 41
¶:¶ ¶:¶ 20
¶now¶ ¶now¶ 42
¶noun¶ ¶noun¶ 20
¶is¶ ¶is¶ 20
¶following¶ ¶following¶ 20
¶player¶ ¶player¶ 20
¶;¶ ¶;¶ 20
¶say¶ ¶say¶ 41
¶"[list of NPC people following player] will attend you.
[list of NPC people not following player] will not."¶ ¶"[list of NPC people following player] will attend you.
[list of NPC people not following player] will not."¶ 20
¶|__¶ ¶|__¶ 0a
¶Understand¶ ¶understand¶ 20
¶"privacy"¶ ¶"privacy"¶ 20
¶as¶ ¶as¶ 20
¶ostracizing¶ ¶ostracizing¶ 20
¶.¶ ¶.¶ 20
¶Ostracizing¶ ¶ostracizing¶ 20
¶is¶ ¶is¶ 20
¶an¶ ¶an¶ 20
¶action¶ ¶action¶ 0a
¶applying¶ ¶applying¶ 20
¶to¶ ¶to¶ 20
¶nothing¶ ¶nothing¶ 20
¶.¶ ¶.¶ 20
¶Carry¶ ¶carry¶ 20
¶out¶ ¶out¶ 20
¶ostracizing¶ ¶ostracizing¶ 20
¶:¶ ¶:¶ 20
¶now¶ ¶now¶ 20
¶every¶ ¶every¶ 0a
¶NPC¶ ¶npc¶ 20
¶person¶ ¶person¶ 20
¶not¶ ¶not¶ 20
¶following¶ ¶following¶ 20
¶player¶ ¶player¶ 20
¶is¶ ¶is¶ 20
¶in¶ ¶in¶ 20
¶a¶ ¶a¶ 20
¶random¶ ¶random¶ 20
¶room¶ ¶room¶ 20
¶which¶ ¶which¶ 0a
¶is¶ ¶is¶ 20
¶not¶ ¶not¶ 20
¶the¶ ¶the¶ 20
¶location¶ ¶location¶ 20
¶;¶ ¶;¶ 20
¶say¶ ¶say¶ 20
¶"[list of every NPC person
not following the player] leaves the room."¶ ¶"[list of every NPC person
not following the player] leaves the room."¶ 20
¶|__¶ ¶|__¶ 0a
¶|__¶ ¶|__¶ 0a
¶|__¶ ¶|__¶ 0a
¶|__¶ ¶|__¶ 0a
